src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/helper/SpringAPIServiceTest.kt

```diff
@@ -44,6 +39,6 @@ import kotlin.annotation.AnnotationTarget.CLASS
 @ActiveProfiles("test")
 @EnableWebFlux
 @EnableWebFluxSecurity
-@SpringBootTest(classes = [WebClientConfiguration::class, WebClientAutoConfiguration::class, OAuth2ClientAutoConfiguration::class, SecurityAutoConfiguration::class, ReactiveOAuth2ClientAutoConfiguration::class])
+@SpringBootTest
```

Comment:
I couldnt find any specific classes to add and get the serialisation of json working, so just went with everything, hope that is ok

Reply:
@mikehalmamoj defo knows what needs to be added here so we don't load every class under the sun

Reply:
ok thanks ill try that